<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSCChangeListCreator</class>
 <widget class="QMainWindow" name="CSCChangeListCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>728</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>728</height>
   </size>
  </property>
  <property name="font">
   <font>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>CSCChangeListCreator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1366</width>
     <height>728</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background-color:rgb(211, 211, 211);
	font-size: 11px;
    font-weight: bold;
}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px;
    padding-left: 5px;
    color:blue
}</string>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QLabel" name="label_wsp">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Workspace</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_server">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>107.113.53.30:1716</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pwd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_server">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>33</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_wsp">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>SVMC_sys_lam.bich_DOC_1716</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_user">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>33</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>User</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_branch">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>//CSC_NILE/Strawberry/EXYNOS5/a5xelte_NN/EUR/OLB/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_go">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>80</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(0, 0,128);
    color: rgb(255,255,255);
}</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_user">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>lam.bich</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_branch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Branch</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>601</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background-color: rgb(178, 225, 255);
	font-size: 11px;
    font-weight: bold;
}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px;
    padding-left: 5px;
    color:blue
}</string>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QLabel" name="label_tag_values">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sales (separated by ;)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tag_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Data directory</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_directory">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>D:\CSC\</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_sale">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>File name</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_file_name">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>keystrings.dat</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="le_sales">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pb_search">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>601</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(0, 0,128);
    color: rgb(255,255,255);
	font-size: 11px;
    font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>601</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background-color:rgb(211, 211, 211);
	font-size: 11px;
    font-weight: bold;
}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px;
    padding-left: 5px;
    color:blue
}</string>
    </property>
    <property name="title">
     <string>Logs</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>581</width>
       <height>361</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>579</width>
        <height>359</height>
       </rect>
      </property>
      <widget class="QTextEdit" name="te_message">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>561</width>
         <height>341</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Version: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_author">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>670</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> lam.bich</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
